### ISSUE TITLE
Can not get marathon logo

### ISSUE THREAD

**Issue by fengyehong** [mesosphere/marathon#1954]
With the following build step, i can not get marathon logo in GUI.
```
git clone --recursive --branch releasing-0.9.1 https://github.com/mesosphere/marathon.git 
cd marathon/
sbt assembly
./bin/build-distribution
```
![2015-08-06 14 34 28](https://cloud.githubusercontent.com/assets/12654661/9105296/f55fedba-3c48-11e5-9157-08f554b33f2e.png)

Browser met 404 when getting /img/marathon-logo.png

In fact, the logo picture is under /dist/img/marathon-logo.png
So i have to copy them to make it work.
```
cd src/main/resources/assets/
cp -r dist/img/ ./
```
Is this part of the standard build step?

**Comment by aldipower** [comment]
Hi @fengyehong . Thanks for mention this.
This should be fixed in further versions, but I will take a look into 0.9.1.

**Comment by aldipower** [comment]
This will be fixed in 0.9.2 release.
https://github.com/mesosphere/marathon/pull/1978
